src/drivers/intel/cavs/dmic.c

```diff
@@ -845,10 +845,12 @@ static int configure_registers(struct dai *dai,
 	}
 
 	/* OUTCONTROL0 and OUTCONTROL1 */
-	of0 = (dmic_prm[0]->fifo_bits == 32) ? 2 : 0;
+	of0 = ((dmic_prm[0]->fifo_bits == 32) ||
+		(dmic_prm[0]->fifo_bits == 24))  ? 2 : 0;
 
 #if DMIC_HW_FIFOS > 1
-	of1 = (dmic_prm[1]->fifo_bits == 32) ? 2 : 0;
+	of1 = ((dmic_prm[1]->fifo_bits == 32) ||
+	 (dmic_prm[1]->fifo_bits == 24)) ? 2 : 0;
```

Comment:
Please align to parenthesis.

Reply:
@mrajwa approved provided you fix @jajanusz comment